Esignet/scripts/Esignet_Test_Script.jmx

```diff
@@ -5,7 +5,7 @@
       <stringProp name="TestPlan.comments"></stringProp>
       <boolProp name="TestPlan.functional_mode">false</boolProp>
       <boolProp name="TestPlan.tearDown_on_shutdown">true</boolProp>
-      <boolProp name="TestPlan.serialize_threadgroups">false</boolProp>
+      <boolProp name="TestPlan.serialize_threadgroups">true</boolProp>
       <elementProp name="TestPlan.user_defined_variables" elementType="Arguments" guiclass="ArgumentsPanel" testclass="Arguments" testname="User Defined Variables" enabled="true">
         <collectionProp name="Arguments.arguments"/>
```

Comment:
Wallet Binding (Execution) - Genarate Binding Public Key, can you keep this before the response assertions ? So it is kind of understand it happens before sending the request. We can also keep above Header manager

Reply:
Done. Have kept it above Header Manager.